circuit_knitting/cutting/qpd/qpd.py

```diff
@@ -1143,7 +1144,13 @@ def _decompose_qpd_instructions(
         for data in inst.operation.definition.data:
             # Can ignore clbits here, as QPDGates don't use clbits directly
             assert data.clbits == ()
-            tmp_data.append(CircuitInstruction(data.operation, qubits=[qubits[0]]))
+            try:
+                equiv = EagleEquivalenceLibrary.get_entry(data.operation)[0]
```

Comment:
The equivalence libraries in CKT are guaranteed to only have, at most, a single equivalence defined for now.